### DISCUSSION TITLE
connect VPN and turn on hotspot is not network

### DISCUSSION THREAD

**Discussion by zll123dxd** [Mygod/VPNHotspot#357] [Q&A]
Sorry to bother you again.

I use a rooted phone with VPN Hotspot installed.
After following operations, my phone has no network :
1.connect to VPN (type : IPSec Xauth PSK/RSA)
2.use VPN hotspot App to turn on hotspot
What puzzles me is that although the mobile phone with the hotspot has no network, the client device connected to the hotspot still has the network .
Also,  client device's network doesn't seem to go through  vpn.

Server Deviecs : xiaomi (android 9)

I think maybe nat in iptables is affecting the normal working of ipsec xauth.
NAT:
>>-A tetherctrl_nat_POSTROUTING -o rmnet_data3 -j MASQUERADE

Not sure if it is correct, please help me.

**Comment by Mygod** [comment]
You should turn on the wlan0 (or whatever it's called) switch in this app.
